examples/Makefile-qpg6095

```diff
@@ -0,0 +1,300 @@
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+.NOTPARALLEL:
+
+TARGET        = arm-none-eabi
+PREFIX        = qpg6095
+CROSS_COMPILE = $(TARGET)-
+
+AR                              = $(CROSS_COMPILE)ar
+CC                              = $(CROSS_COMPILE)gcc
+CXX                             = $(CROSS_COMPILE)g++
+LINK                            = $(CROSS_COMPILE)ld
+OBJCOPY                         = $(CROSS_COMPILE)objcopy
+OBJDUMP                         = $(CROSS_COMPILE)objdump
+SIZE                            = $(CROSS_COMPILE)size -A
+NM                              = $(CROSS_COMPILE)nm
+
+BuildJobs                      ?= 9
+
+configure_OPTIONS                                = \
+    --enable-cli                                   \
+    --enable-diag                                  \
+    --enable-ftd                                   \
+    --enable-mtd                                   \
+    --with-examples=$(PREFIX)                      \
+    MBEDTLS_CPPFLAGS="$(QPG6095_MBEDTLS_CPPFLAGS)" \
+    $(NULL)
+
+ifeq ($(CLI_LOGGING),1)
+configure_OPTIONS              += --enable-cli-logging
+endif
+
+QPG6095_CONFIG_FILE_CPPFLAGS  = -DOPENTHREAD_PROJECT_CORE_CONFIG_FILE='\"openthread-core-qpg6095-config.h\"'
+QPG6095_CONFIG_FILE_CPPFLAGS += -I$(PWD)/examples/platforms/qpg6095/
+QPG6095_MBEDTLS_CPPFLAGS      = -DMBEDTLS_CONFIG_FILE='\"mbedtls-config.h\"'
+QPG6095_MBEDTLS_CPPFLAGS     += -DMBEDTLS_USER_CONFIG_FILE='\"qpg6095-mbedtls-config.h\"'
+QPG6095_MBEDTLS_CPPFLAGS     += -I$(PWD)/examples/platforms/qpg6095/crypto
+QPG6095_MBEDTLS_CPPFLAGS     += -I$(PWD)/third_party/mbedtls
+QPG6095_MBEDTLS_CPPFLAGS     += -I$(PWD)/third_party/mbedtls/repo/include
+QPG6095_MBEDTLS_CPPFLAGS     += -I$(PWD)/third_party/mbedtls/repo/include/mbedtls
+
+
+COMMONCFLAGS                     := \
+    -fdata-sections                 \
+    -ffunction-sections             \
+    -O2                             \
+    $(QPG6095_CONFIG_FILE_CPPFLAGS) \
+    -W                              \
+    -Wall                           \
+    $(NULL)
+#    -Os                           \
+#    -g                            \
+
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_LOG_OUTPUT=OPENTHREAD_CONFIG_LOG_OUTPUT_APP
```

Comment:
I believe this is not necessary and already handled here: https://github.com/openthread/openthread/blob/master/examples/common-switches.mk#L38